site/content/docs/5.3/content/tables.md

```diff
@@ -243,6 +249,11 @@ Behind the scenes it looks like this:
 
 Add `.table-sm` to make any `.table` more compact by cutting all cell `padding` in half.
 
+<!-- Boosted mod : design callout -->
+{{< ods-alert info >}}
+These compact tables are recommended by Orange Design System.
```

Comment:
Seems a bit weird when you consult directly this section. I'd propose more context so that this content can be consulted independently.

```suggestion
To display basic tables, Orange Design System recommends using these compact tables.
```

Something like this.